### PR TITLE
feat(authenticator): l10n Chinese support

### PR DESCRIPTION
**Pull request by quirkeer**
*Description of changes:*

Add l10n Chinese translation.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by dnys1** [comment]
Thanks for your PR, @quirkeer! Would you be willing to run the `tool/generate_l10n.sh` script and verify that the app looks and feels correct when using Chinese? There may be more changes necessary on our end and I'd love your feedback if so!

File: packages/amplify_authenticator/lib/src/l10n/src/titles/titles_zh.arb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "@@locale": "zh",
+    "@@context": "amplify_authenticator:titles",
+    "signin": "登陆你的账户",
```
Comment (HuiSF) [inline packages/amplify_authenticator/lib/src/l10n/src/titles/titles_zh.arb:4]:
```suggestion
    "signin": "登录你的账户",
```

File: packages/amplify_authenticator/lib/src/l10n/src/countries/countries_zh.arb [hunk 2]
```diff
@@ -0,0 +1,250 @@
+{
+  "@@locale": "zh",
+  "@@context": "amplify_authenticator:countries",
+  "selectDialCode": "选择您的国家区号",
```
Comment (HuiSF) [inline packages/amplify_authenticator/lib/src/l10n/src/countries/countries_zh.arb:4]: Translation of possessive determiner is inconsistent, e.g. "您的" vs "你的".

**Review by HuiSF — COMMENTED** [review]
Thanks for contributing!

**Comment by HuiSF** [comment]
Hello @quirkeer 
Currently, developers can provide and use their own localization for authenticator  ([reference](https://ui.docs.amplify.aws/components/authenticator?platform=flutter#internationalization-i18n)). We will be investigating possibility of enabling internationalization in the Authenticator based on App running device locale. Hence, I'm going to close this PR for now.

Thank you very much again for contributing. Please feel free to follow up if you have any questions and suggestions.

**Comment by dJani97** [comment]
An updated link to the docs: https://ui.docs.amplify.aws/flutter/connected-components/authenticator/customization

May I ask why such community PRs are rejected? The documented way seems to involve overriding 7+ resolver classes and hundreds of methods, each returning a single localization value. Is this really the only way to localize the library?

Having premade ARB files contributed by the community would be so much easier for everyone.

Amplify-UI already does this: https://github.com/aws-amplify/amplify-ui/tree/main/packages/ui/src/i18n/dictionaries/authenticator. Plus, creating your own localization is a breeze, just copy one of these dictionaries and change the values. Can we have something like that?

**Pull request closed without merging**